eureka/niriss_class.py

```diff
@@ -0,0 +1,516 @@
+import os
```

Comment:
If this file is going to contain nothing other than Stage 3 code, it should be moved to S3_data_reduction

Reply:
Also, too much of this code is already written in other functions. You should be calling those other functions instead of duplicating code